pycode/epidemiology/epidata/getDIVIData.py

```diff
@@ -47,380 +47,40 @@
 import os
 import sys
 import bisect
-from datetime import timedelta, date
-import pandas
+from datetime import timedelta, date, datetime
+import pandas as pd
 
 from epidemiology.epidata import getDataIntoPandasDataFrame as gd
 from epidemiology.epidata import defaultDict as dd
 from epidemiology.epidata import geoModificationGermany as geoger
 from epidemiology.epidata import modifyDataframeSeries
 
 
-def adjust_data(df, date_of_data):
-    """!Adjusts data such that the data structure looks the same for all dates
+def cut_of_dates(df, start_date, end_date):
```

Comment:
This is okay for now. But this is a prototype for a function that could be called for any kind of date specific data frame. So it should be moved outside getDiviData with the next issue.

Furthermore, a name like "extract_subframe_based_on_dates" would be better suited.